documentation/src/css/custom.css

```
@@ -1,95 +1,286 @@
+  text-shadow: 2px 2px 8px rgba(0, 0, 0, 0.6);
+  text-decoration: none !important;
+}
+
+.hero__subtitle {
+  font-size: 1.5rem;
+  color: var(--color-red-lighter);
+  margin-bottom: 2rem;
+  max-width: 600px;
+  margin-left: auto;
+  margin-right: auto;
+  color: #f8f9fa !important;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

color is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)